lib/config.yml

```diff
@@ -19,21 +19,48 @@ Rails:
 Bundler/OrderedGems:
   Enabled: false
 
+Layout/AlignParameters:
+  EnforcedStyle: with_fixed_indentation
+
 Layout/CaseIndentation:
   EnforcedStyle: end
 
 Layout/ElseAlignment:
   Enabled: false
 
+Layout/EmptyLinesAroundBeginBody:
+  Enabled: false
+
 Layout/EmptyLinesAroundBlockBody:
   Enabled: false
 
+Layout/EmptyLinesAroundClassBody:
+  Enabled: false
+
 Layout/EmptyLinesAroundExceptionHandlingKeywords:
   Enabled: false
 
+Layout/FirstArrayElementLineBreak:
+  Enabled: true
+
+Layout/FirstHashElementLineBreak:
+  Enabled: true
+
+Layout/FirstMethodArgumentLineBreak:
```

Comment:
I agree in the example you gave. It looks off and we shouldn't allow that case. I disagree in the examples from the cop. I think the cop example is more of a common case where the line runs to long. 
Funny enough the cop would ignore your example if you left off the parens.